front/app/containers/Admin/messaging/texting/components/RemainingCharacters.tsx

```diff
@@ -9,7 +9,7 @@ interface Props {
 const RemainingCharacters = ({ remainingChars, overCharacterLimit }: Props) => {
   const getText = () => {
     if (overCharacterLimit) {
-      return <>{remainingChars * -1} over the character limit</>;
+      return <>{remainingChars * -1} characters over the limit</>;
```

Comment:
What's proper English here? 😁

Reply:
I would say `1 character over the limit` or `3 characters over the limit` and remove the preceding `-`. I don't think it really matters as it would still be clear with the red text, and I really doubt anyone would actually make this mistake, but `-1 characters over the limit` could be seen as a double negative.

Reply:
It actually is a positive number. If overCharacterLimit is true, it means remainingChars is a negative number. So then times -1 gives us a positive number. Need to adjust for singular/plural though.

Reply:
Never mind then, that's a clever solution. In that case just the singular/plural like you mentioned